feature/experimental/policy/otg_tests/prefix_set_test/prefix_set_test.go

```diff
@@ -100,45 +108,47 @@ func TestPrefixSet(t *testing.T) {
 
 func TestPrefixSetWithCLIConfig(t *testing.T) {
```

Comment:
if cli is no longer used, then 
a) this test should be renamed
b) readme needs an update once again.